migrations/versions/20181220_populate_measures.py

```diff
@@ -158,6 +158,8 @@ def upgrade():
         f"not match subtopic_measure join table ({count_subtopic_measures})"
     )
 
+    op.execute("DROP TABLE max_measure_version")
```

Comment:
This shouldn't be needed as temporary tables will be dropped by Postgres at the end of the session anyway?  And this migration has already run anyway?

Reply:
I explained this in the commit message: let me know if it's clear enough.


```When alembic applies database migrations, it does so with a single
wrapping transaction (by default), which means that temporary tables
persist across individual migrations. Therefore, to keep allow multiple
migrations to use a temporary table either as part of a set of
migrations going in at once, or by itself, we need to drop the temporary
table at the end of each migration that uses it (and create it in any
subsequent migrations that need it).```

Reply:
That's good, clear and makes sense (and I did wonder if it was this when I saw the second one). I just hadn't dug into individual commits, sorry.

Makes me wonder if there's a way to make Alembic run each migration in its own transaction by default, which might be preferable.